### ISSUE TITLE
Add media symbols to granite accel parse

### ISSUE THREAD

**Issue by danrabbit** [elementary/granite#312]
"Audio play" ►
"Audio pause" ⏸
"Audio stop" ⏹
"Audio next" ⏭
"Audio previous" ⏮ 
"Eject" ⏏

I can't seem to find suitable non-color versions of volume icons in unicode. Maybe there isn't :grimacing: 

"Audio raise volume" 🔊
"Audio lower volume" 🔉
"Audio mute" 🔇

**Comment by cassidyjames** [comment]
I like the idea but yeah the full-color emoji feel wrong here. Would it be wild to support images instead of just text so we can use symbolics? Otherwise, I don't know if we can force the characters to be rendered in the default font and not Noto Color.

**Comment by danrabbit** [comment]
Yeah, we could use icons in the view shown in the screenshot but since this function is also used for tooltips (which can't use icons), then we'd have to do it in switchboard not granite. I think at least using the few unicode symbols that aren't color here would be good for more consistency

**Comment by cassidyjames** [comment]
@danrabbit ah yeah I forgot this just returns a string so it can be used in tooltips. I agree the non-color symbols are great.
